### DISCUSSION TITLE
Windows 11 22H2 - any ETA?

### DISCUSSION THREAD

**Discussion by mopi74** [AveYo/MediaCreationTool.bat#209] [Q&A]
Dear AveYo,
I have seen many users these days asking about your upcoming update supporting Win 11 22H2 but I haven't seen any clear answer from you... I respectfully think your users deserve a bit more consideration, don't you?

Thanks

**Comment by dajoey** [comment]
Deserve is a strong word.  I also am antsy for an update, but until I start paying their bills, I deserve nothing from them.

**Reply by mopi74** [reply]
Dear dajoey,
thanks for your reply but I respectfully disagree, I was simply asking for a simple answer, a 10 second matter with no costs for anyone... if not I don't understand one thing: who this repository has been created for?
